src/charging-station/ChargingStationWorker.ts

```diff
@@ -28,11 +41,90 @@ if (ChargingStationUtils.workerPoolInUse()) {
   }
 }
 
+class TEMP {
+  public hashId: string;
+  public command: CommandCode;
+  public connectorId: number;
+  public idTag: string | null;
+}
+
+let station: ChargingStation;
```

Comment:
Given the way it's affected, in workerSet mode, it will reference the latest created CS in the worker. 
How do you guarantee that stop/startTransaction will be triggered on all CS instances?